src/components/MedicationReports.js

```diff
@@ -0,0 +1,61 @@
+import React from "react"
+
+import { darkgrey } from "../colors"
+import { observer } from "mobx-react"
+import { DateTime } from "luxon"
+import Fold from "../primitives/Fold"
+
+import { Icon } from "@mdi/react"
+import { mdiPill } from "@mdi/js"
+import Heading from "../primitives/Heading"
+import { Paragraph } from "reakit";
+
+import styled from "styled-components"
+
+const MedicationReports = observer(({ store }) => (
+      <div>
```

Comment:
This line (and the following) are indented too much - can you de-dent to two spaces?